app/assets/javascripts/discourse/components/invite-panel.js.es6

```diff
@@ -180,6 +182,10 @@ export default Component.extend({
     );
   },
 
+  showContactPicker: Ember.computed(function() {
```

Comment:
would be better to use `computed` directly here, didn't do it when I gave you this example.

```
import { computed } from "@ember/object";

showContactPicker: computed(function() {
  return this.capabilities.hasContactPicker;
});
```